crowdsec-docs/docs/appsec/quickstart.md

```diff
@@ -51,26 +68,155 @@ You can now restart CrowdSec:
 sudo systemctl restart crowdsec
 ```
 
-### Setup the remediation component
+### First test
+
+Before moving on with the Remediation Component, let's ensure that everything we already setup works as expected.
+
+1. Create a Remediation Component (Bouncer) API Key:
 
-Now that our Application Security Component is running in CrowdSec, we need to set up the remediation component to interact with it.
 
-Note that every remediation component may have different option names for this.
-I suggest you check directly in the remediation component documentation, but here we will use the [OpenResty remediation component](/u/bouncers/openresty) as an example.
+```bash
+sudo cscli bouncers add test_waf -k this_is_a_bad_password
+API key for 'test_waf':
+
+   this_is_a_bad_password
+
+Please keep this key since you will not be able to retrieve it!
+```
+
+2. Emit a legitimate request to the AppSec Component:
+
+```bash
+curl -X POST localhost:7422/ -i -H 'x-crowdsec-appsec-uri: /test' -H 'x-crowdsec-appsec-ip: 42.42.42.42' -H 'x-crowdsec-appsec-host: foobar.com' -H 'x-crowdsec-appsec-verb: POST' -H 'x-crowdsec-appsec-api-key: this_is_a_bad_password'
+```
 
-To set up the AppSec in the OpenResty remediation component, we just need to set the Application Security Component URL previously exposed:
+Which will give us an answer such as:
+
+```bash
+HTTP/1.1 200 OK
+Date: Tue, 30 Jan 2024 15:43:50 GMT
+Content-Length: 36
+Content-Type: text/plain; charset=utf-8
+
+{"action":"allow","http_status":200}
+```
+
+3. Emit a malevolent request to the Appsec Component:
+
+:::info
+We're trying to access a `.env` file, a [common way to get access to some credentials forgotten by a developer.](https://app.crowdsec.net/hub/author/crowdsecurity/appsec-rules/vpatch-env-access)
+:::
+
+```bash
+curl -X POST localhost:7422/ -i -H 'x-crowdsec-appsec-uri: /.env' -H 'x-crowdsec-appsec-ip: 42.42.42.42' -H 'x-crowdsec-appsec-host: foobar.com' -H 'x-crowdsec-appsec-verb: POST' -H 'x-crowdsec-appsec-api-key: this_is_a_bad_password'
+
+```
+
+Our request is detected and blocked by the AppSec Component:
+
+```bash
+HTTP/1.1 403 Forbidden
+Date: Tue, 30 Jan 2024 15:57:08 GMT
+Content-Length: 34
+Content-Type: text/plain; charset=utf-8
+
+{"action":"ban","http_status":403}
+```
 
-```bash title="/etc/crowdsec/bouncers/crowdsec-openresty-bouncer.conf"
+Let's now delete our test API Key:
+
+```bash
+sudo cscli bouncers delete test_waf
+```
```

Comment:
Within the `quick start` guide should we care about showing this, could this be a separate section? since if we making it `noob` friendly they might not know how to use curl or even understand what we are doing

Reply:
yeah I get you. I make this section expandable and mark it as optional, gud ?